internal/pkg/runtime/c/lib/util/capability.c

```diff
@@ -1,13 +1,16 @@
-/* 
+/*
  * Copyright (c) 2017-2018, SyLabs, Inc. All rights reserved.
  * Copyright (c) 2017, SingularityWare, LLC. All rights reserved.
- * 
+ *
  * This software is licensed under a 3-clause BSD license.  Please
  * consult LICENSE.md file distributed with the sources of this project regarding
  * your rights to use or distribute this software.
- * 
+ *
  */
 
+
+#ifdef LINUX_CAPABILITY_H
```

Comment:
LINUX_CAPABILITY_H is defined in the `mlocal/checks/project-post.chk` based on ability to compile program that includes `<linux/capability.h>`.